Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
 def customiseDataRun2Common_25ns(process):
     process = customiseDataRun2Common(process)
 
+    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
+    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
```

Comment:
no, HcalRemoveAddSevLevel is not defined at this point:

NameError: global name 'HcalRemoveAddSevLevel' is not defined

I'm trying another way (I've used in my private test)



On Fri, 25 Sep 2015, Slava Krutelyov wrote:

> 
> In Configuration/DataProcessing/python/RecoTLR.py:
> 
> > @@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
> >  def customiseDataRun2Common_25ns(process):
> >      process = customiseDataRun2Common(process)
> > 
> > +    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
> > +    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
> 
> I suppose just
> HcalRemoveAddSevLevel.AddFlag(process.hcalRecAlgos,"HFDigiTime",8) will work
> 
> —
> Reply to this email directly or view it on GitHub.[AEx02rSXOrKFOn2XFeBBP6Fd5qCNbvz-ks5o1TYDgaJpZM4GDoQK.gif]
> 
> 
>